src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```
@@ -5,6 +5,7 @@
     <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
     <EnableDefaultItems>true</EnableDefaultItems>
     <Nullable>annotations</Nullable>
+    <DefineConstants>$(DefineConstants),NO_SUPRESS_GC_TRANSITION</DefineConstants>
```

Should we use **;** instead of **,**?

Edit: found also a typo 

```suggestion
    <DefineConstants>$(DefineConstants);NO_SUPPRESS_GC_TRANSITION</DefineConstants>
```

---

; vs , is fine, since Roslyn accepts both. We have various of both in the repo so I dint have an strong opinion on that.